starknet-foundry/crates/forge/src/lib.rs

```diff
@@ -169,15 +179,21 @@ fn collect_tests_from_file(
 
 #[allow(clippy::implicit_hasher)]
 pub fn run(
-    input_path: &Utf8PathBuf,
+    package_path: &Utf8PathBuf,
+    lib_path: &Utf8PathBuf,
     linked_libraries: &Option<Vec<LinkedLibrary>>,
     runner_config: &RunnerConfig,
     corelib_path: Option<&Utf8PathBuf>,
     contracts: &HashMap<String, StarknetContractArtifacts>,
     predeployed_contracts: &Utf8PathBuf,
 ) -> Result<Vec<TestFileSummary>> {
```

Comment:
A this point this method could use some documentation (`/// comment`)

Reply:
You can create an issue for that.

Reply:
#263